eng/Version.Details.xml

```diff
@@ -9,289 +9,289 @@
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>0b951c16de0f39e13cce8372e11c28eb90576662</Sha>
+      <Sha>0e82b48369290d71f976d96f8880ed24a710a0b2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.ParameterDefaultValue.Sources" Version="3.0.0-rc2.19463.5" CoherentParentDependency="Microsoft.EntityFrameworkCore">
+    <Dependency Name="Microsoft.Extensions.ParameterDefaultValue.Sources" Version="3.0.1-servicing.19506.5" CoherentParentDependency="Microsoft.EntityFrameworkCore">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>0b951c16de0f39e13cce8372e11c28eb90576662</Sha>
+      <Sha>0e82b48369290d71f976d96f8880ed24a710a0b2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Primitives" Version="3.0.0" CoherentParentDependency="Microsoft.EntityFrameworkCore">
+    <Dependency Name="Microsoft.Extensions.Primitives" Version="3.0.1" CoherentParentDependency="Microsoft.EntityFrameworkCore">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>0b951c16de0f39e13cce8372e11c28eb90576662</Sha>
+      <Sha>0e82b48369290d71f976d96f8880ed24a710a0b2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.TypeNameHelper.Sources" Version="3.0.0-rc2.19463.5" CoherentParentDependency="Microsoft.EntityFrameworkCore">
+    <Dependency Name="Microsoft.Extensions.TypeNameHelper.Sources" Version="3.0.1-servicing.19506.5" CoherentParentDependency="Microsoft.EntityFrameworkCore">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>0b951c16de0f39e13cce8372e11c28eb90576662</Sha>
+      <Sha>0e82b48369290d71f976d96f8880ed24a710a0b2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.ValueStopwatch.Sources" Version="3.0.0-rc2.19463.5" CoherentParentDependency="Microsoft.EntityFrameworkCore">
+    <Dependency Name="Microsoft.Extensions.ValueStopwatch.Sources" Version="3.0.1-servicing.19506.5" CoherentParentDependency="Microsoft.EntityFrameworkCore">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>0b951c16de0f39e13cce8372e11c28eb90576662</Sha>
+      <Sha>0e82b48369290d71f976d96f8880ed24a710a0b2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.WebEncoders" Version="3.0.0" CoherentParentDependency="Microsoft.EntityFrameworkCore">
+    <Dependency Name="Microsoft.Extensions.WebEncoders" Version="3.0.1" CoherentParentDependency="Microsoft.EntityFrameworkCore">
       <Uri>https://github.com/aspnet/Extensions</Uri>
-      <Sha>0b951c16de0f39e13cce8372e11c28eb90576662</Sha>
+      <Sha>0e82b48369290d71f976d96f8880ed24a710a0b2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Internal.Extensions.Refs" Version="3.0.0-rc2.19463.5" CoherentParentDependency="Microsoft.EntityFrameworkCore">
+    <Dependency Name="Microsoft.Internal.Extensions.Refs" Version="3.0.0-rc2.19463.5" CoherentParentDependency="Microsoft.EntityFrameworkCore" Pinned="true">
```

Comment:
@mmitche does this need to move down to the bottom of the `<ProductDependencies>` element with Microsoft.NETCore.App.Ref. And, should the `CoherentParentDependency` attribute be removed.

Reply:
It can be removed, and can be moved to the end, though i do not think it matters (I don't think there are direct dependencies on efcore downstream of aspenetcore)